applications/NXmonopd.nxdl.xml

```diff
@@ -69,7 +80,7 @@
                 <field name="polar_angle" type="NX_FLOAT" axis="1">
                   <doc>where ndet = number of detectors</doc>
```

Comment:
Mismatch

Reply:
Being unfamiliar with NXmonopd, should I just remove the doc string? and for mismatch are you saying that the size of _polar_angle_ should not be _nDet_?

Reply:
I meant symbol mismatched so should be nDet but the doc also duplicates the definition in the symbol table.

Reply:
I think you should remove the doc string. The presented symbol definition is now correctly provided in the symbol table, so deleting the doc string doesn't remove information from the application definition.

Reply:
ok I will delete it